client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx

```diff
@@ -8,14 +8,14 @@ import styles from './BatchChangesListIntro.module.scss'
 export const BatchChangesChangelogAlert: React.FunctionComponent = () => (
     <DismissibleAlert
         className={styles.batchChangesListIntroAlert}
-        partialStorageKey="batch-changes-list-intro-changelog-3.30"
+        partialStorageKey="batch-changes-list-intro-changelog-3.31"
     >
         <div className={classNames(styles.batchChangesListIntroCard, 'card h-100 p-2')}>
             <div className="card-body">
-                <h4>New Batch Changes features in version 3.30</h4>
+                <h4>New Batch Changes features in version 3.31</h4>
                 <ul className="mb-0 pl-3">
                     <li>
-                        Changesets can now be published from the UI!{' '}
+                        Changesets can now be published from the preview UI.{' '}
```

Comment:
Adam pointed out to me "preview UI" may not carry meaning to users, so maybe:
```suggestion
                        Changesets can now be set to published when previewing new or updated batch changes.{' '}
```